cs-amazon/src/main/java/io/cloudslang/content/amazon/utils/OutputsUtil.java

```diff
@@ -33,4 +52,34 @@ private OutputsUtil() {
 
         return signatureReturnResultMap;
     }
+
+    public static void putResponseIn(Map<String, String> queryMapResult) {
+        if (queryMapResult != null) {
+            if (queryMapResult.containsKey(STATUS_CODE) && (valueOf(SC_OK).equals(queryMapResult.get(STATUS_CODE))) && queryMapResult.containsKey(RETURN_RESULT) && !isEmpty(queryMapResult.get(RETURN_RESULT))) {
+                queryMapResult.put(RETURN_CODE, SUCCESS);
+            } else {
+                queryMapResult.put(RETURN_CODE, FAILURE);
+            }
+        } else {
+            queryMapResult = new HashMap<>();
+            queryMapResult.put(EXCEPTION, "Null response!");
+            queryMapResult.put(RETURN_CODE, FAILURE);
+            queryMapResult.put(RETURN_RESULT, "The query returned null response!");
+        }
+    }
+
+    public static void extractResponseAsResult(Map<String, String> queryMapResult, String outputName, String xPathQuery) {
```

Comment:
In my opinion, this kind of methods that modify a passed parameter should have a sugestive name like the one above "putResponseIn" or they should return the object, in this case 'queryMapResult'. This way you avoid confusion when reading the code wehere this method is being used.